tools/kcp-installer/kcp.Dockerfile

```
@@ -1,5 +1,5 @@
 # The base version of the Kyma Operator that will be used to build Control Plane Installer
-ARG INSTALLER_VERSION="ef024a2c"
+ARG INSTALLER_VERSION="98e02519"
```

I think we can use the newer version here
```suggestion
ARG INSTALLER_VERSION="4260fe58"
```